### ISSUE TITLE
Using Devise for Huginn user Authentication

### ISSUE THREAD

**Issue by ismail44** [cantino/huginn#1297]
I see that Devise (and Warden) are being used in the application; presumably for 3rd-party agent authentication. Is it possible to use Devise for logging users into the GUI? 

Thanks.

**Comment by cantino** [comment]
Devise is the primary authentication system being used.  It controls how you log in with username and password.  Could you clarify what you're trying to do?

**Comment by ismail44** [comment]
Yeah, I am seeing that... I actually worded what I was wanting to ask
incorrectly. I mean to ask about using OmniAuth (with Devise) for
authentication.

So, what I want to do is authenticate users with OpenAM. When a user comes
into the app, they will already have a session token set by OpenAM, which i
need to parse and either create the user (if they do not exist), or find
the user (and update some things) and that's it. I see that OmniAuth is
being used (presumably for agent authentication), I was wondering it was
possible to leverage what's there for the agent authentication into what
Devise is doing for the app user authentication.

Sorry about the initial flub-up.

**Comment by cantino** [comment]
Gotcha.  It shouldn't be too hard, but isn't wired yet.  See #176 and #1081.  We'd love for someone to contribute this change!
